Dockerfile

```diff
@@ -28,17 +25,14 @@ WORKDIR /app/${package}
 
 # Run tests and return coverage analysis
 RUN python -m coverage run tests/test_${package}.py \
- && python -m coverage html -d /htmlcov --omit=/usr/local/*
+&& python -m coverage html -d /htmlcov --omit=/usr/local/*
```

Comment:
What is the container which this produces ultimately used for?  In particular, do you really want the coverage data included in the final output?

Reply:
I believe this is correct yes ^^

Reply:
Dustin and I did a little investigation:  the coverage file is generated in the first phase of the container build; it is propagated to the final container image; the CI later pulls the image from Quay, creates a container from it (but doesn't start it running), pulls coverage file from it, and then deletes the container; the CI then processes and uploads the coverage data as an artifact so that it can be viewed later.

Thus, there is no compelling need for the coverage file to be in the final container image, other than as a mechanism for making it available to CI; and, the desire to minimize the size of the container image is motivation for _not_ including it in the final image.

Instead, the initial build phase should export the coverage file (either by writing it to a mapped directory, or by having the CI extract it after the image is built, as it does with the final image, today), so that we don't propagate it to the final image.  The CI can then process and upload the artifact.

Reply:
Yeah, I remember noticing that when I was working on the Pbench plugin, but it seemed to low in the noise at the time to worry about. So this is good progress...